### ISSUE TITLE
Add option to be non-interactive.

### ISSUE THREAD

**Issue by gitlost** [wp-cli/package-command#62]
See https://github.com/wp-cli/package-command/pull/60#pullrequestreview-85985229

Composer supports being run non-interactively so it could be useful to have an option to set this.

In particular it would avoid getting prompted for one's ssh password as can happen depending on one's authorization settings, which is problematic for scripts.

**Comment by schlessera** [comment]
Yes, agree, we can simply forward such an option. It's an easy addition.
